packages/mariadb-connection-module/src/index.js

```diff
@@ -0,0 +1,15 @@
+const knex = require('knex');
+const databaseConfig = require('../src/config/knexfile');
+const DB = knex(databaseConfig);
+const logger = require('./utils/logger');
+
+DB.raw('select 1+1 as result')
+    .then(() => {
+        logger.info(`[pl-mariadbconnection-module]: Mongoose connection is opened to ${databaseConfig.connection.host}`)
+    })
+    .catch(err => {
+        logger.error(`[pl-mariadbconnection-module]: Connection error : ${err.message}`);
+        process.exit(1)
+    })
```

Comment:
What is the purpose of doing a SQL statement at this part of the flow? because if the connection isn't working, Knex by default returns an exception that should be handled by a `try-catch` statement.

There are multiple ways to achieve this. One of the most common ones is the following:
```suggestion
const logger = require('./utils/logger');
let dbConnection;
try {
 logger.info(`[pl-mariadbconnection-module]: Oppening a db connection to ${databaseConfig.connection.host}`);
 if (!dbConnection) {
    dbConnection = knex(databaseConfig);
  }
} catch(err) {
       logger.error(`[pl-mariadbconnection-module]: Connection error : ${err.message}`);
       process.exit(1);
}

module.exports = dbConnection;
```

Reply:
Thank you, You're right